client/src/components/WorkspacePanels/AudioEntry.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import { Draggable } from "react-beautiful-dnd";
+
+const useAudio = (url) => {
+  const [audio] = useState(new Audio(url));
+  const [playing, setPlaying] = useState(false);
+
+  const toggle = () => setPlaying(!playing);
```

Comment:
Do you think we should capitalize this as well just like the `Loading` function below?